consensus/pacemaker.go

```diff
@@ -185,17 +191,16 @@ func (p *paceMaker) RestartTimer() {
 		select {
 		case <-ctx.Done():
 			if ctx.Err() == context.DeadlineExceeded {
-				p.consensusMod.nodeLog(typesCons.PacemakerTimeout(p.consensusMod.CurrentHeight(), p.consensusMod.step, p.consensusMod.round))
-				p.InterruptRound()
+				p.InterruptRound("timeout")
 			}
 		case <-clock.After(stepTimeout + 30*timePkg.Millisecond): // Adding 30ms to the context timeout to avoid race condition.
```

Comment:
is there any specific reason to add exactly 30ms, or it is added because it is just enough to avoid race condition?